tests/e2e/event_handler_appsync/test_appsync_resolvers.py

```diff
@@ -110,3 +110,41 @@ def test_appsync_get_related_posts_batch(appsync_endpoint, appsync_access_key):
         assert post["post_id"] in related_posts_ids
     for post in data["getPost"]["relatedPostsAsync"]:
         assert post["post_id"] in related_posts_ids
+
+
+@pytest.mark.xdist_group(name="event_handler")
+def test_appsync_get_related_posts_batch_with_aggregate(appsync_endpoint, appsync_access_key):
```

Comment:
could you break the assertions for sync first, then async, and add more details on GIVEN/WHEN as we're doing two tests here, plz?

Reply:
Done